R/pkg/R/sparkR.R

```diff
@@ -31,20 +31,27 @@ connExists <- function(env) {
 #' Stop the Spark context.
 #'
 #' Also terminates the backend this R session is connected to
+#' @export
 sparkR.stop <- function() {
+  .Deprecated("sparkR.session.stop")
+  sparkR.session.stop()
```

Comment:
It could - my preference would be to match the "init" and "stop" naming so if we are to change `sparkR.init` we could change `sparkR.stop` too